libraries/api-documenter/src/MarkdownRenderer.ts

```diff
@@ -115,12 +115,12 @@ export class MarkdownRenderer {
 
   private static _getEscapedText(text: string): string {
```

Comment:
>_getEscapedText [](start = 17, length = 15)

It would be nice to have a test case for it.

Reply:
Good suggestion.  I added this case to the existing unit test, and found a couple bugs in doing so.  :-)

Reply:
Good suggestion.  I added this case to the existing unit test, and found a couple bugs in doing so.  :-)

---
In reply to: [144613075](https://github.com/Microsoft/web-build-tools/pull/393#discussion_r144613075) [](ancestors = 144613075)